bigbluebutton-html5/imports/ui/components/media/service.js

```diff
@@ -1,5 +1,11 @@
+import React from 'react';
 import Presentations from '/imports/api/2.0/presentations';
 import { isVideoBroadcasting } from '/imports/ui/components/screenshare/service';
+import iosService from '/imports/ui/services/ios-handler/index';
+import PresentationAreaContainer from '../presentation/container';
```

Comment:
What a mess.

Since when our `service.js` started importing containers? This part is for communicating with the server-side only.

This has to be changed, all you are supposed to do here is to fetch and pass the data from the collections together with the callbacks for the communication with the api.